### DISCUSSION TITLE
zh/docs/how-to-connect/saml/tencent-cloud/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#3137] [Docs comments]
# zh/docs/how-to-connect/saml/tencent-cloud/

使用Casdoor作为SAML IdP

https://casdoor.org/zh/docs/how-to-connect/saml/tencent-cloud/

**Comment by tiny-v** [comment]
我有个问题， 如果在腾讯云那儿， 一个Idp对应多个角色。 但在 Casdoor 上， 怎样指定用户和其中某一个，或多个角色绑定。

**Comment by tiny-v** [comment]
I hava one question. If we bind multiple roles to one idp in tencent cloud.   In the casdoor side,  is there any way to bind one of these roles to one casdoor user,,   but not bind all the roles in default.

**Comment by Pooua** [comment]
Casdoor version: v2.165.0 does not have a "type" option.
